01_fib/saurabhvyas9/main_test.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
```

Comment:
Nitpick: I'd like to see testing of the actual `fib()` function, not just the main output in general

ie. one test for a positive number, one for a negative, one for (say) zero (extra credits if you consider what happens with overflows!)

Also means if you (eg.) broke your negafibonacci later, you'd actually know you broke that functionality rather than just "Computer Says Error"

Reply:
Yes I agree with that.
One single test case seems not quite enough.
Add corner cases, test `fib` proper.

Reply:
Done.